holidayextras/ui-component-tile
# Adding product schema

#### What does this PR do?
- Adds product schema to a product tile.
- This PR assumes a tile will always be associated with a product, for now that is true. In the future it may not.

#### What unit or integration tests does this PR have?
- Adds a test for title

#### What selenium tests does this PR have?
N/A

#### How should a developer review this?
Go to http://schema.org/Product and look at the example product schema
Ensure the same process is followed on this element

#### How should this be manually tested?
One you have checked out the branch
`npm install`
`npm run build-dev`
and open `dev/index.html` to view the component
run the resulting html through https://developers.google.com/structured-data/testing-tool/ and ensure it passes the test

#### Any background context you want to provide?
UI components must work for everyone, therefore SEO and Schema are important.

#### What are the relevant tickets?
None yet

#### What gif best describes how you feel about this work?
![tile fall](http://media.giphy.com/media/5fXJmIyVrIkNy/giphy.gif)

---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
